Documentation/Books/Manual/Views/ArangoSearch/README.md

```diff
@@ -1,54 +1,293 @@
-# ArangoSearch views powered by IResearch
+# Bringing the power of IResearch to ArangoDB
 
-ArangoSearch is a natively integrated AQL extension making use of the
-IResearch library.
+## What is ArangoSearch
 
-ArangoSearch allows one to:
+ArangoSearch is a natively integrated AQL extension making use of the
+[IResearch library](https://github.com/iresearch-toolkit/iresearch).
 
+Arangosearch allows one to:
 * join documents located in different collections to one result list
-* filter documents based on AQL boolean expressions and functions
-* sort the result set based on how closely each document matched the filter
+* search documents based on AQL boolean expressions and functions
+* sort the result set based on how closely each document matched the search condition
 
 A concept of value 'analysis' that is meant to break up a given value into
 a set of sub-values internally tied together by metadata which influences both
-the filter and sort stages to provide the most appropriate match for the
+the search and sort stages to provide the most appropriate match for the
 specified conditions, similar to queries to web search engines.
 
 In plain terms this means a user can for example:
-
 * request documents where the 'body' attribute best matches 'a quick brown fox'
 * request documents where the 'dna' attribute best matches a DNA sub sequence
 * request documents where the 'name' attribute best matches gender
-* etc. (via custom analyzers)
+* etc... (via custom analyzers described in the next section)
```

Comment:
Can we change this to something like the following?

```md
In plain terms this means a user can request documents where:

* the 'body' attribute best matches 'a quick brown fox'
* the 'dna' attribute best matches a DNA sub sequence
* the 'name' attribute best matches gender
* and much more via custom [analyzers](Analyzers.md)
```

`described in the next section` - such phrases should be avoided in general. If content is added this may no longer be true, and the next section is not even about custom analyzers already. If at all, use anchor links to the respective page/section.